test/cypress/integration/group-paying.spec.js

```diff
@@ -475,11 +475,14 @@ describe('Group Payments', () => {
     ])
 
     cy.log('user3 receives a notification for the payment and clicking on it opens a "Payment details" modal.')
+    // cy.getByDT('DOESNOTEXIST')
```

Comment:
If the PR is no longer a draft, then this commented-out debug stuff should be removed (along with the one below)